applications/volume_rendering_xr/main.cpp

```diff
@@ -169,58 +285,74 @@ class App : public holoscan::Application {
   }
 
   const std::string render_config_file_;
+  const std::string write_config_file_;
   const std::string density_volume_file_;
   const std::string mask_volume_file_;
+  const bool enable_eye_tracking_;
 };
 
 int main(int argc, char** argv) {
-  const std::string render_config_file_default(
-      "/workspace/holoscan-openxr/data/volume_rendering/config.json");
-  const std::string density_volume_file_default(
-      "/workspace/holoscan-openxr/data/volume_rendering/highResCT.mhd");
-  const std::string mask_volume_file_default(
-      "/workspace/holoscan-openxr/data/volume_rendering/smoothmasks.seg.mhd");
-
-  std::string render_config_file;
+  // TODO: Apparent issue with parsing non-default args in HoloHub
```

Comment:
Do you know which issue this is?

Reply:
Ah, determined this issue was due to a missing arg, the config filepath was not previously passed to `XrTransformRenderOp` where it was expected. Fix is applied in this update, will remove this comment and accompanying debugging statements.